package.json

```diff
@@ -47,7 +47,7 @@
     "validate-commit-message": "^3.0.1"
   },
   "peerDependencies": {
-    "@dcl/schemas": ">=1.1.0"
+    "@dcl/schemas": ">=3.1.1"
   },
```

Comment:
If something is a dependency it should not be also a peerDependency, should be moved to devDependencies to use the peerDependency

Reply:
jajaja, yes I know, but I didn't want to make that change because I don't know if that package should be `peerDependency` or not. Can we create a separate issue to fix it?

Reply:
@menduz